### DISCUSSION TITLE
two arguments for report flag needed

### DISCUSSION THREAD

**Discussion by jkleinlercher** [aquasecurity/trivy#4768] [Documentation]
### Description

after changing report flag in https://github.com/aquasecurity/trivy/pull/2186 the new behaviour should be changed also in https://github.com/aquasecurity/trivy/blob/main/docs/tutorials/kubernetes/cluster-scanning.md

### Link

https://github.com/aquasecurity/trivy/blob/main/docs/tutorials/kubernetes/cluster-scanning.md

### Suggestions

the report flag needs two arguments now. in https://github.com/aquasecurity/trivy/blob/main/docs/tutorials/kubernetes/cluster-scanning.md there is just one argument documented.

**Comment by knqyf263** [comment]
@chen-keinan Please take a look.

**Comment by chen-keinan** [comment]
@jkleinlercher could you please elaborate more on what is missing? the two report flags are supported and produce different output

**Reply by jkleinlercher** [reply]
@chen-keinan in https://github.com/aquasecurity/trivy/blob/main/docs/tutorials/kubernetes/cluster-scanning.md there is the command

`trivy k8s --report=summary `

mentioned. However, with this command you get an error:

` FATAL   Require at least 1 argument`

The reason is, the "trivy k8s" command needs at least an argument "all" or "cluster", or some other resource type. The command is correctly described in https://github.com/aquasecurity/trivy/blob/main/docs/docs/target/kubernetes.md but in https://github.com/aquasecurity/trivy/blob/main/docs/tutorials/kubernetes/cluster-scanning.md it is missing the argument.

**Reply by itaysk** [reply]
fixed by https://github.com/aquasecurity/trivy/pull/4815
